# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../etcnet/0002-redo-BOOTPROTO-separator-match-in-locale-insensitive.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 670cedacc62048bbfa8e46d58d27786d67170870 Mon Sep 17 00:00:00 2001
From: Sergey Bolshakov <sbolshakov@altlinux.org>
Date: Sun, 17 Jan 2010 17:52:03 +0300
Subject: [PATCH 2/4] redo BOOTPROTO separator match in locale-insensitive manner

---
 etc/net/scripts/config-ipv4      |    6 +++---
 etc/net/scripts/ifdown           |    4 ++--
 etc/net/scripts/ifdown-ifplugd   |    2 +-
 etc/net/scripts/ifdown-removable |    2 +-
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/etc/net/scripts/config-ipv4 b/etc/net/scripts/config-ipv4
index c4c0b9d..ce6dc0e 100755
--- a/etc/net/scripts/config-ipv4
+++ b/etc/net/scripts/config-ipv4
@@ -33,13 +33,13 @@ case $ACTION in
 				ipv4ll)
 					try_ipv4ll && config_ipv4_routes_rules add
 					;;
-				dhcp[\ -,]static)
+				dhcp[-,\ ]static)
 					try_dhcp || try_static && config_ipv4_routes_rules add
 					;;
-				dhcp[\ -,]ipv4ll)
+				dhcp[-,\ ]ipv4ll)
 					try_dhcp || try_ipv4ll && config_ipv4_routes_rules add
 					;;
-				dhcp[\ -,]ipv4ll[\ -,]static)
+				dhcp[-,\ ]ipv4ll[-,\ ]static)
 					try_dhcp || try_ipv4ll || try_static && config_ipv4_routes_rules add
 					;;
 				*)
diff --git a/etc/net/scripts/ifdown b/etc/net/scripts/ifdown
index 60db2df..76ccac1 100755
--- a/etc/net/scripts/ifdown
+++ b/etc/net/scripts/ifdown
@@ -59,7 +59,7 @@ xargise_file $MYIFACEDIR/ipneigh "$IP neigh del dev $NAME"
 
 # Look if there is DHCP agent hanging around.
 case "$BOOTPROTO" in
-	dhcp|dhcp[\ -,]static|dhcp[\ -,]ipv4ll|dhcp[\ -,]ipv4ll[\ -,]static)
+	dhcp|dhcp[-,\ ]static|dhcp[-,\ ]ipv4ll|dhcp[-,\ ]ipv4ll[-,\ ]static)
 		stop_dhcp_client
 	;;
 	*)
@@ -68,7 +68,7 @@ esac
 
 # Kill ipv4ll agent, if any.
 case "$BOOTPROTO" in
-	ipv4ll|dhcp[\ -,]ipv4ll|dhcp[\ -,]ipv4ll[\ -,]static)
+	ipv4ll|dhcp[-,\ ]ipv4ll|dhcp[-,\ ]ipv4ll[-,\ ]static)
 		stop_ipv4ll_client
 	;;
 	*)
diff --git a/etc/net/scripts/ifdown-ifplugd b/etc/net/scripts/ifdown-ifplugd
index b03320b..86e9a05 100755
--- a/etc/net/scripts/ifdown-ifplugd
+++ b/etc/net/scripts/ifdown-ifplugd
@@ -46,7 +46,7 @@ fi
 xargise_file $MYIFACEDIR/ipneigh "$IP neigh del dev $NAME"
 
 case "$BOOTPROTO" in
-	dhcp|dhcp[\ -,]static|dhcp[\ -,]ipv4ll[\ -,]static)
+	dhcp|dhcp[-,\ ]static|dhcp[-,\ ]ipv4ll[-,\ ]static)
 		stop_dhcp_client
 	;;
 	*)
diff --git a/etc/net/scripts/ifdown-removable b/etc/net/scripts/ifdown-removable
index cfe38a8..c631fab 100755
--- a/etc/net/scripts/ifdown-removable
+++ b/etc/net/scripts/ifdown-removable
@@ -87,7 +87,7 @@ fi
 is_yes "$IFDOWN_CHILDREN" && ifdown_children
 xargise_file $MYIFACEDIR/ipneigh "$IP neigh del dev $NAME"
 case "$BOOTPROTO" in
-	dhcp|dhcp[\ -,]static|dhcp[\ -,]ipv4ll|dhcp[\ -,]ipv4ll[\ -,]static)
+	dhcp|dhcp[-,\ ]static|dhcp[-,\ ]ipv4ll|dhcp[-,\ ]ipv4ll[-,\ ]static)
 		stop_dhcp_client
 	;;
 	*)
-- 
1.6.6.2

